### PR TITLE
chore: simplify function names

### PR DESCRIPTION
**Pull request by tenthirtyam**
### Summary

Changes the following non-exported function names:

- `Write-PowerManagementLogMessage` >> `Write-LogMessage`
- `Debug-CatchWriterForPowerManagement` >> `Write-DebugMessage`

### Type

- [ ] Bugfix
- [ ] Enhancement or Feature
- [ ] Code Style or Formatting
- [ ] Documentation
- [ ] Refactoring
- [x] Chore
- [ ] Other
        Please describe:

### Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### Test and Documentation

- [x] Tests have been completed.
- [ ] Documentation has been added or updated.

### CODE REVIEW

**Review by GaryJBlake — COMMENTED** [review]
`Write-LogMessage` is an exported function of `PowerVCF`, will this cause an issue ?

**Comment by tenthirtyam** [comment]
I need to verify, but a non-exported function is supposed to takes precedence within its own module, ensuring no conflicts with similarly named functions from another module, even if exported.

File: SampleScripts/PowerManagement-WorkloadDomain.ps1
Comment (joisika) [inline SampleScripts/PowerManagement-WorkloadDomain.ps1:1]: Please change `Write-PowerManagementLogMessage` to `Write-LogMessage` in the whole file

Comment (tenthirtyam) [reply SampleScripts/PowerManagement-WorkloadDomain.ps1:1]: Resolved in https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-power-management/commit/f0c8669a13b0c39b36db82cdbc2d20f24523bd95.

**Pull request closed without merging**